openassessment/xblock/static/js/src/studio/oa_edit_settings.js

```diff
@@ -255,6 +302,16 @@ OpenAssessment.EditSettingsView.prototype = {
         isValid = (this.startDatetimeControl.validate() && isValid);
         isValid = (this.dueDatetimeControl.validate() && isValid);
         isValid = (this.leaderboardIntField.validate() && isValid);
+        if (this.fileUploadType() != 'custom') {
```

Comment:
NTH: Can we flip this check ? By checking else part first, will make it more readable.